hermes_nemisis/tests/test_calibration.py

```diff
@@ -24,13 +28,22 @@ def level1_file(tmp_path_factory):
         pass
     return fn
 
+# def test_readbin_CCSDS():  
+#     output_file = readbin_CCSDS('hermes_nemisis/data/hermes_NEM_l0_2024085-174022_v00.bin')
+#     # assert output_file.is_file()  # this fails.  output_file is a string object, not a file. is this a HermesData issue?
```

Comment:
If the `readbin_CCSDS.py` file is not needed, can you remove these commented out test cases? If it is needed, then you should un-comment the test case and validate that they pass.